engine/gamesys/src/gamesys/components/comp_particlefx.cpp

```diff
@@ -105,23 +104,18 @@ namespace dmGameSystem
         world->m_ConstantBuffers.SetSize(max_emitter_count);
         memset(world->m_ConstantBuffers.Begin(), 0, sizeof(dmRender::HNamedConstantBuffer)*max_emitter_count);
 
-        uint32_t buffer_size = dmParticle::GetVertexBufferSize(ctx->m_MaxParticleCount, dmParticle::PARTICLE_GO);
+        // position   : 3
+        // color      : 4
+        // texcoord0  : 2
+        // page_index : 1
+        const uint32_t default_vx_size = sizeof(float) * (3 + 4 + 2 + 1);
+        const uint32_t buffer_size     = ctx->m_MaxParticleCount * 6 * default_vx_size;
+        world->m_VertexBufferData.SetCapacity(ctx->m_MaxParticleCount * 6 * default_vx_size);
```

Comment:
We don't have any particle "formats" now, the size of the vx buffer is only dependent on the vertex size now.